app/containers/Search/index.js

```diff
@@ -202,15 +202,24 @@ export class Search extends React.PureComponent { // eslint-disable-line react/p
       title: intl.formatMessage(appMessages.buttons.printTitle),
       icon: 'print',
     }];
-    const availableSettings = settings
-      .keySeq()
-      .filter((key) => (settings.getIn([key, 'available'])))
-      .toList();
```

Comment:
immutable supports filtering maps (opposed to JS objects), also no need to convert to list